enterprise_gateway/services/processproxies/processproxy.py

```diff
@@ -512,7 +524,10 @@ def receive_connection_info(self):
                 while 1:
                     buffer = conn.recv(1024)
                     if not buffer:  # send is complete
-                        connect_info = json.loads(data)
+                        self.log.debug("Received Payload '{}'".format(data))
+                        payload = self._decrypt(data)
+                        self.log.debug("Decrypted Payload '{}'".format(payload))
+                        connect_info = json.loads(payload)
```

Comment:
Let's keep these in for now.  Thinking this might produce a good amount of duplicate info (that's already duplicated itself).